tests/test_timing_model.py

```diff
@@ -385,6 +394,60 @@ def test_parfile_and_timfile_jumps(timfile_jumps):
     assert fs[4] == "1"
 
 
+class TOAOrderSetup:
+    parfile = os.path.join(datadir, "NGC6440E.par")
+    timfile = os.path.join(datadir, "NGC6440E.tim")
```

Comment:
Is this a single-telescope file? I think it's just GBT data? It should be fine but it would be nice to have a mixed-telescope data set to ensure we catch weirdness with treating things differently by telescope.

Reply:
I agree in principle, but the existing test data don't have lots of multi-telescope data and those that are there (1713) are very large.  I could possibly put in a new one.

Reply:
OK, have a new test `test_toa_shuffle.py` which does the same routines but with multi-telescope multi-frequency simulated data.